src/calendar/tests/CalendarMonth.spec.js

```diff
@@ -0,0 +1,232 @@
+import React from 'react/addons';
+import CalendarMonth from '../CalendarMonth';
+import CalendarDate from '../CalendarDate';
+import moment from 'moment';
+import {} from 'moment-range';
+import _ from 'underscore';
+
+
+const TestUtils = React.addons.TestUtils;
+
+describe('The CalendarMonth Component', function () {
+
+  const getCalendarMonth = (props) => {
+
+    props = _.extend({
+      firstOfWeek: 0,
+      firstOfMonth: this.firstOfMonth,
+      enabledRange: moment.range(moment(), moment().add(3, 'years')),
+      dateComponent: CalendarDate,
+      disableNavigation: false,
+      dateRangesForDate: function () {
+        return {
+          count: function () {
+            return props.count || 1;
+          },
+          getIn: function (data) {
+            if (data[0] === 0) {
+              return '#333';
+            }
+            return '#444';
+          },
+        };
+      },
+      onMonthChange: function () {},
+      onYearChange: function () {},
+    }, props);
+
+
+    return (<CalendarMonth {...props} />);
+  };
+
+  const useShallowRenderer = (props) => {
+    this.shallowRenderer = TestUtils.createRenderer();
+    this.shallowRenderer.render(getCalendarMonth(props));
+    this.renderedComponent = this.shallowRenderer.getRenderOutput();
+    this.container = this.renderedComponent.props.children[0];
+    this.table = this.renderedComponent.props.children[1];
+  };
+
+  const useDocumentRenderer = (props) => {
+    this.renderedComponent = TestUtils.renderIntoDocument(getCalendarMonth(props));
+  };
+
+  beforeEach(() => {
+    this.spyCx = spyOn(CalendarMonth.prototype.__reactAutoBindMap, 'cx').and.callFake( (data) => {
```

Comment:
Correct me if I'm wrong; But does this not mean we're binding to the describe object - rather than Jasmine's context object that gets reset at the end of each test - as described [here](http://jasmine.github.io/2.0/introduction.html#section-The_<code>this</code>_keyword) ?